src/Helsenorge.Messaging/ServiceBus/ServiceBusCore.cs

```diff
@@ -304,9 +304,11 @@ private async Task Send(ILogger logger, IMessagingMessage message)
             // update some properties on the cloned message
             clonedMessage.To = await ConstructQueueName(logger, originalMessage.FromHerId, QueueType.Error).ConfigureAwait(false); // change target 
             clonedMessage.TimeToLive = Settings.Error.TimeToLive;
-            clonedMessage.FromHerId = originalMessage.ToHerId;
-            clonedMessage.ToHerId = originalMessage.FromHerId;
-            
+            var fromHerId = originalMessage.FromHerId;
+            var toHerId = originalMessage.ToHerId;
+            clonedMessage.FromHerId = fromHerId;
+            clonedMessage.ToHerId = toHerId;
```

Comment:
Have we validated that this does not affect other parts in a negative way?

Reply:
I think what I'm trying to say is why are not flipping the values like this anymore?
```csharp
var fromHerId = originalMessage.ToHerId;
var toHerId = originalMessage.FromHerId;
 ```

Reply:
according to the definitions of fromHerId and toHerId they are mandatory in all messages. and describe "who sent the message" and "who is the receiver".  And they should not have anything to do with the Error itself. When A send B a message, B finds an error an report back to A. Then it will be (more) correct to swap them. and then this part of the PR will be wrong they way I see it.
If the receiver of errormessage want to know more of the message causing the problem it can use the 'originalMessageId' to find the correct message.

- another thing to think about is that the error with this message might be the toHerId. in those cases its not correct to map them as above either. it should be the actual herid of the sending part...

Reply:
I did not mean to change the original order - the point was it did not work since they were references in the real implementation